spikeextractors/extractors/spikeglxrecordingextractor/spikeglxrecordingextractor.py

```diff
@@ -41,7 +41,12 @@ def __init__(self, file_path, x_pitch=21, y_pitch=20):
         self._meta = meta
 
         # Traces in 16-bit format
-        self._raw = makeMemMapRaw(self._npxfile, meta)  # [chanList, firstSamp:lastSamp+1]
+        if '.cbin' in self._npxfile.name: # compressed binary format used by IBL
+            import mtscomp
```

Comment:
Can you have a try except here in case `mtscomp` is not installed?

Reply:
done